build.gradle

```diff
@@ -78,3 +73,5 @@ publishing {
 wrapper {
     gradleVersion = '6.0.1'
 }
+tasks.register("noOp")
+defaultTasks "noOp"
```

Comment:
Hope that doesn't bother anyone. By default gradle invokes `help` task and that unnecessarily bloats console output